components/layout/Header.vue

```diff
@@ -636,6 +840,142 @@ export default {
+                            background: url("/landing/header-menu/platform-overview-bg.svg");
+                            transform: rotateX(180deg);
+                            top: 0;
+                            left: 0;
+                        }
+
+                        .dropdown-item {
+                            padding: 0;
+                            white-space: unset;
+                            pointer-events: auto !important;
+                            display: flex;
+                            flex-direction: column;
+                            gap: calc($spacer * 2.7);
+                            position: relative;
+                            z-index: 1;
+
+                            .material-design-icon, span, p {
+                                color: $white !important;
+                            }
+                            &:hover {
+                                background-color: transparent;
+                            }
+                        }
+
+                        @include media-breakpoint-down(lg) {
+                            border-radius: calc($spacer * 0.25);
+                            .dropdown-item {
+                                white-space: unset;
+                            }
+                        }
+                        li {
+                            .submenu-img {
+                                padding: calc($spacer * 1.8) calc($spacer * 0.4);
+                                min-height: 113px;
+                                min-width: 100%;
+                                position: relative;
+
+                                p {
+                                    font-size: $spacer;
+                                    font-style: italic;
+                                    font-weight: 800;
+                                    color: $white;
+                                }
+
+                                img {
+                                    position: absolute;
+                                    left: 8%;
+                                }
+                            }
+                        }
+                    }
+                }
+
+                .solutions-menu {
+                    gap: 1rem;
+                    left: -280%;
+                    max-width: calc($spacer * 59);
+
+                    .dropdown-column {
+                        display: flex;
+                        min-width: calc($spacer * 18);
+                        flex-direction: column;
+                        gap: $spacer;
+
+                        @include media-breakpoint-down(lg) {
+                            margin-bottom: calc($spacer * 2);
+                        }
+
+                        .dropdown-item {
+                            white-space: unset;
+                        }
+
+                        .column-caption {
+                            font-size: $font-size-base;
+                            font-weight: 600;
```

Comment:
We have no variable for differents font weight ?

Reply:
no we are not using variable for font-weight